gateleen-queue/src/main/resources/gateleen_queue_splitter_configuration_schema

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": {
+    "$ref": "#/definitions/QueueSplitter"
+  },
+  "definitions": {
```

Comment:
Try to add more schema constraints like `oneOf` to clearify that only postfixFromStatic **or** (postfixFromHeader or postfixFromUrl) can be used. Checkout the schema defined for `gateleen-delegates` where this is used.